web/massastation/src/i18n/en_US.json

```diff
@@ -6,6 +6,15 @@
     "mystation": {
       "loading": "Loading...",
       "browse": "Browse the store below and manage plugins you've installed in this section"
+    },
+    "sidepanel": {
+      "sidepanel-banner": "I want to create and install my own module !",
+      "sidepanel-description": "How to do it: {url}",
+      "sidepanel-title": " Manual installation",
+      "sidepanel-subtitle": "Copy the link of the module zip",
+      "sidepanel-placeholder": "URL",
+      "sidepanel-fe-error": "Please enter a valid url",
+      "sidepanel-be-error": "Failed to upload plugin"
```

Comment:
Since we are under `sidepanel` already, no need to repeat the word `sidepanel`.

Otherwise we end up: `sidepanel. sidepanel-..` all we need is `sidepanel.something.else...`